src/main/java/org/broadinstitute/hellbender/tools/walkers/coverage/DepthOfCoverage.java

```diff
@@ -0,0 +1,456 @@
+    private boolean omitPartiallyCoveredGenes = false;
+
+    /**
+     * Output file format (e.g. csv, table, rtable); defaults to r-readable table.
+     */
+    @Argument(fullName = "output-format", doc = "The format of the output file", optional = true)
+    CoverageOutputWriter.DEPTH_OF_COVERAGE_OUTPUT_FORMAT outputFormat = CoverageOutputWriter.DEPTH_OF_COVERAGE_OUTPUT_FORMAT.CSV;
+
+
+    // ---------------------------------------------------------------------------
+    //
+    // Advanced arguments
+    //
+    // ---------------------------------------------------------------------------
+
+    /**
+     * Normally, sites where the reference is N (or another non-canonical base) are skipped. If this option is enabled, these sites will be included in DoC calculations if there is coverage from neighboring reads.
+     */
+    @Advanced
+    @Argument(fullName = "include-ref-n-sites", doc = "Include sites where the reference is N", optional = true)
+    private boolean includeRefNBases = false;
+    /**
+     * Sets the low-coverage cutoff for granular binning. All loci with depth < START are counted in the first bin.
+     */
+    @Advanced
+    @Argument(fullName = "start", doc = "Starting (left endpoint) for granular binning", optional = true, minValue = 0)
+    private int start = 1;
+    /**
+     * Sets the high-coverage cutoff for granular binning. All loci with depth > STOP are counted in the last bin.
+     */
+    @Advanced
+    @Argument(fullName = "stop", doc = "Ending (right endpoint) for granular binning", optional = true, minValue = 1)
+    private int stop = 500;
+    /**
+     * Sets the number of bins for granular binning
+     */
+    @Advanced
+    @Argument(fullName = "nBins", doc = "Number of bins to use for granular binning", optional = true, minValue = 0, minRecommendedValue = 1)
+    private int nBins = 499;
+
+    /**
+     * This option simply disables writing separate files for per-sample summary statistics (total, mean, median, quartile coverage per sample). These statistics are still calculated internally, so enabling this option will not improve runtime.
+     */
+    @Argument(fullName = "omit-per-sample-stats", doc = "Do not output the summary files per-sample", optional = true)
+    private boolean omitSampleSummary = false;
+    /**
+     * By default, coverage is partitioning by sample, but it can be any combination of sample, readgroup and/or library.
+     */
+    @Argument(fullName = "partition-type", shortName = "pt", doc = "Partition type for depth of coverage", optional = true)
+    private EnumSet<DoCOutputType.Partition> partitionTypes = EnumSet.of(DoCOutputType.Partition.sample);
+
+    /**
+     * Consider a spanning deletion as contributing to coverage. Also enables deletion counts in per-base output.
+     */
+    @Advanced
+    @Argument(fullName = "include-deletions", shortName = "dels", doc = "Include information on deletions", optional = true)
+    private boolean includeDeletions = false;
```

Comment:
Could the default be switched to include deletions as contributing to coverage as opposed to not?